style.css

```diff
@@ -1 +1,304 @@
-/*  Add your styling here */
+@import url(https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&family=Roboto:wght@100&display=swap);
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
```

Comment:
Is the large number of columns justified for this design?
Looks like we have a couple of empty rows in the grid.

![Uploading image.png…]()